src/App.tsx

```diff
@@ -1,34 +1,161 @@
-/* eslint-disable max-len */
 import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { getTodos, getUser } from './api';
+
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
-export const App: React.FC = () => {
-  return (
-    <>
-      <div className="section">
-        <div className="container">
-          <div className="box">
-            <h1 className="title">Todos:</h1>
+type State = {
+  todosFromServer: Todo[],
+  user: User | null,
+  currentTodo: Todo | null,
+  isDataReady: boolean,
+  isClickedOnTodos: boolean,
+  query: string,
+  option: string,
+};
 
-            <div className="block">
-              <TodoFilter />
-            </div>
+export class App extends React.Component {
```

Comment:
it would be better to choose one style for the whole application (class based components or functional)